tubular/nominal.py

```diff
@@ -471,8 +472,20 @@ class MeanResponseTransformer(BaseNominalTransformer, BaseMappingTransformMixin)
     """Transformer to apply mean response encoding. This converts categorical variables to
     numeric by mapping levels to the mean response for that level.
 
+    For a continuous or binary response there is an natural mean to calculate and the columns
```

Comment:
I think this could be explained in simpler terms given it is the default usage.

'For a continuous or binary response the categorical columns specified will have values replaced with the mean response for each category.'